src/sentry/workflow_engine/tasks.py

```diff
@@ -0,0 +1,73 @@
+from sentry.issues.status_change_consumer import group_status_update_registry
+from sentry.issues.status_change_message import StatusChangeMessageData
+from sentry.models.activity import Activity
+from sentry.models.group import Group
+from sentry.silo.base import SiloMode
+from sentry.tasks.base import instrumented_task
+from sentry.taskworker import config, namespaces, retry
+from sentry.types.activity import ActivityType
+from sentry.utils import metrics
+
+SUPPORTED_ACTIVITIES = [ActivityType.SET_RESOLVED]
+
+
+@instrumented_task(
+    name="sentry.workflow_engine.processors.process_workflow_task",
+    queue="process_workflows",
+    default_retry_delay=5,
+    max_retries=5,
+    soft_time_limit=50,
+    time_limit=60,
+    silo_mode=SiloMode.REGION,
+    taskworker_config=config.TaskworkerConfig(
+        namespace=namespaces.workflow_engine_tasks,
+        processing_deadline_duration=60,
+        retry=retry.Retry(
+            times=3,
+            delay=5,
+        ),
+    ),
+)
+def process_workflow_task(activity_id: int, detector_id: int) -> None:
+    """
+    Process a workflow task identified by the given Activity ID and Detector ID.
+
+    This task will retry up to 3 times with a delay of 5 seconds between attempts.
+    It has a soft time limit of 50 seconds and a hard time limit of 60 seconds.
+
+    The task will get the Activity from the database, create a WorkflowEventData object,
+    and then process the data in `process_workflows`.
+    """
```

Comment:
Any ideas on what kind of throughput tasks in the workflow_tasks namespace will have?

Reply:
We can add this to the alerts taskworker pools when we get US deployed. In other regions throughput will be small enough that it isn't a concern.